biblat_schema/models.py

```diff
@@ -0,0 +1,111 @@
+from mongoengine import *
```

Comment:
Falta especificar la codificación
```
# -*- coding: utf-8 -*-
```
https://www.python.org/dev/peps/pep-0263/#defining-the-encoding